tests/operators/test_update.py

```diff
@@ -63,29 +65,39 @@ def operator(
     else:
         smaps = None
     kspace_locs = kspace_locs.astype(np.float32)
-    return get_operator(backend)(
-        kspace_locs,
-        shape,
-        n_coils=n_coils,
-        smaps=smaps,
-        n_batchs=n_batch,
-        n_trans=n_trans,
-        squeeze_dims=False,
-        density=density,
-    )
+    op_args = {
+        "samples": kspace_locs,
+        "shape": shape,
+        "n_coils": n_coils,
+        "n_batchs": n_batch,
+        "squeeze_dims": False,
+        "density": density,
+        "smaps": smaps,
+    }
+    if backend in ["cufinufft"]:
+        op_args.update([("smaps_cached", smaps_cached)])
```

Comment:
```suggestion
        op_args["smaps_cached"] = smaps_cached
```

Reply:
I really thought this when I saw the code, but i thought its some "safe" way to do it perhaps xD

Reply:
when you have a lot of update do to , this might be more efficient (I guess). But here we don't need that, just stay simple.